### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by YiGaZh** [Mojang/bedrock-samples#54]
Hi, is it possible to see examples of more items such a s tridents and fishing rods

**Comment by 9124ah** [comment]
i need too

**Comment by MKokeshi** [comment]
Yes, it is possible to see examples of items like tridents and fishing rods in the **bedrock-samples** repository. This repository typically contains scripts, behavior packs, resource packs, and other examples that demonstrate how to create or modify items, mobs, and gameplay mechanics in Minecraft Bedrock Edition.

### Finding Examples in the Repository

1. **Behavior Packs and Resource Packs:**
   - The **bedrock-samples** repository often includes behavior packs (`.behavior_pack`) and resource packs (`.resource_pack`). These packs define how items behave and how they are rendered in the game. 
   - Look for files within folders like `items`, `loot_tables`, `recipes`, or `functions` to find examples of items such as tridents and fishing rods.

2. **Item Definition Files:**
   - Items like tridents and fishing rods would typically have a JSON file in the behavior pack that defines their properties. For example, tridents may have a file like `trident.json` which defines their behavior, damage, and interaction with the game world.

3. **Search for Specific Item Examples:**
   - You can use the search function in the repository (if browsing online) to look for specific items. Searching for "trident" or "fishing_rod" might bring up files where these items are defined or used.

### Creating Custom Items

If you're looking to create or modify these items, here is a basic outline of how you might do it:

1. **Trident Example:**
   - In a typical JSON definition for a trident, you might see something like this:

   ```json
   {
       "format_version": "1.10",
       "minecraft:item": {
           "description": {
               "identifier": "minecraft:trident",
               "category": "equipment"
           },
           "components": {
               "minecraft:damage": {
                   "damage_chance": 0.2
               },
               "minecraft:throwable": {
                   "do_summon_projectile": true
               }
           }
       }
   }
   ```

   - This file would be found in the `items` folder of a behavior pack and defines how the trident works, including whether it can be thrown and its damage properties.

2. **Fishing Rod Example:**
   - For a fishing rod, the JSON might include how the item interacts with entities and water blocks:

   ```json
   {
       "format_version": "1.10",
       "minecraft:item": {
           "description": {
               "identifier": "minecraft:fishing_rod",
               "category": "equipment"
           },
           "components": {
               "minecraft:use_duration": 32,
               "minecraft:max_damage": 64,
               "minecraft:cooldown": 5,
               "minecraft:use_animation": "pull"
           }
       }
   }
   ```

   - This would also be located in the `items` folder, and it specifies how the fishing rod is used, including cooldowns and durability.

### Summary

In summary, while the **bedrock-samples** repository may not directly include every item by name, it usually provides the necessary frameworks and examples to create or modify items such as tridents and fishing rods. By exploring the relevant JSON files and scripts within the repository, you can find or create the definitions you need for these items.

**Comment by obscuredc** [comment]
@MKokeshi while your explanation is helpful, as of the current push there are no examples of tools (ie. axe, pickaxe, shovel, hoe) nor weapons (melee, ie. sword, trident and ranged, ie. bow, crossbow). The Behavior Pack/Items folder appears to only hold food-related items and a few miscellaneous items that can't really exemplify the behaviors of the weapons the original posters were probably looking for :(
